nuclei/CVE-2021-26855-5890.yaml

```diff
@@ -0,0 +1,32 @@
+id: cve-2021-26855
+info:
+  name: Exchange Server SSRF Vulnerability
+  author: madrobot
+  severity: critical
+  description: |
+    Microsoft Exchange Server Remote Code Execution Vulnerability This CVE ID is unique from CVE-2021-26412, CVE-2021-26854, CVE-2021-26857, CVE-2021-26858, CVE-2021-27065, CVE-2021-27078.
```

Comment:
is this the correct check for cve-2021-26857 because they are saying "his CVE ID is unique from"

Reply:
Changed description for more clarity.